x-pack/platform/plugins/shared/streams/server/routes/streams/management/route.ts

```diff
@@ -102,8 +102,32 @@ export const getStreamsStatusRoute = createServerRoute({
   },
 });
 
+export const getClassicStreamsStatusRoute = createServerRoute({
+  endpoint: 'GET /api/streams/_classic_status',
+  options: {
+    access: 'internal',
```

Comment:
internal routes are usually prefixed with `/internal/` instead of `/api/`. is there a reason to prefix with api here ?